client/reader/reading-time/index.jsx

```diff
@@ -14,7 +14,8 @@ class ReadingTime extends React.PureComponent {
 		if ( timeInMinutes > 1 ) {
 			approxTime = (
 				<span className="reading-time__approx">
-					( { this.props.translate( '~%d min', {
+					({' '}
```

Comment:
It would be nice if we could get rid of these `{' '}` whitespace kludges. In this case, maybe by extracting the big `translate` expression to a variable outside the JSX.